packages/components/src/RecurringSelect/components/MonthlyDayOfWeekSelect/MonthlyDayOfWeekSelect.css

```diff
@@ -0,0 +1,37 @@
+.parentContainer {
+  /* useless, only here to avoid linting error */
+  display: block;
```

Comment:
this css crime is here because I need a selector high enough that it can know how many child components (divs) it has

in order to apply the class name I have to have a `className={styles.parentContainer}` 

in order to have a `.parentContainer` in the css, I can't leave it blank so I'm adding a completely useless piece of code to appease the linter :(((((((((((((((

Reply:
looking at this now, I care for it even less

if we can suppress a stylelint violation, I'd prefer that honestly

or if we can come up with a better overall solution that involves none of this, that also works

Reply:
If we can do something like my idea in the comment below, would that solve this issue of relying on a parent wrapper?

Reply:
line 11 would add it back to the `d.ts` no? Have you tried removing it?